MvcApp/MvcApp/Areas/Users/Views/Posts/Create.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@model MvcApp.Models.Post
```

Comment:
ViewModel orqali ishlash kerak. Dilshod yoki Shoyusufdan so'ren

Reply:
Buni to`g`riladim